File: python_examples/AverageUpdated.py
def averageupdated(source, args):
    if args.CurrentAverage < args.TotalAverages:
        print('\rAverage:', args.CurrentAverage, " / ", args.TotalAverages, end='')
    else:
        print('\rAverage:', args.TotalAverages, " / ", args.TotalAverages, end='')
        print('')

File: matlab_examples/AverageUpdated.m
function AverageUpdated(src, event)   
    average = int32(event.CurrentAverage);  
    lineLength = fprintf('Average: %d/%d', average, int32(event.TotalAverages));
    pause(0.001);
    if (average < int32(event.TotalAverages))
        fprintf(repmat('\b', 1, lineLength));
    else
        fprintf('\n');
    end           
end